ui_test/POM_hiway_asg/pages/hiway_homepage.py

```diff
@@ -0,0 +1,12 @@
+from POM_hiway_asg.pages.constant_var import timesheet_tab_btn
+import time
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') standard import "import time" should be placed before "from POM_hiway_asg.pages.constant_var import timesheet_tab_btn" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=Pylint%3AC0411)